From 432bbf979ee66ee29bb92e35fd6e3ffb948563e3 Mon Sep 17 00:00:00 2001
From: Lee Duncan <lduncan@suse.com>
Date: Wed, 29 Sep 2021 11:48:16 -0700
Subject: [PATCH] Remove dependences from iscsi-init.service

Since iscsid.service depends on it but disables
default dependencies, iscsi-init.service must
also disable default dependencies, or a dependency
loop can be created.
---
 etc/systemd/iscsi-init.service | 1 +
 1 file changed, 1 insertion(+)

diff --git a/etc/systemd/iscsi-init.service b/etc/systemd/iscsi-init.service
index e058ff0..eab4ff2 100644
--- a/etc/systemd/iscsi-init.service
+++ b/etc/systemd/iscsi-init.service
@@ -1,6 +1,7 @@
 [Unit]
 Description=One time configuration for iscsi.service
 ConditionPathExists=!/etc/iscsi/initiatorname.iscsi
+DefaultDependencies=no
 Before=iscsid.service
 
 [Service]
-- 
2.33.0

